vector/src/main/java/im/vector/app/features/onboarding/ftueauth/FtueAuthLegacyStyleCaptchaFragment.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.onboarding.ftueauth
+
+import android.os.Parcelable
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import com.airbnb.mvrx.args
+import im.vector.app.databinding.FragmentLoginCaptchaBinding
+import im.vector.app.features.onboarding.OnboardingAction
+import im.vector.app.features.onboarding.OnboardingViewState
+import im.vector.app.features.onboarding.RegisterAction
+import kotlinx.parcelize.Parcelize
+import javax.inject.Inject
+
+@Parcelize
+data class FtueAuthLegacyStyleCaptchaFragmentArgument(
+        val siteKey: String
+) : Parcelable
+
+/**
+ * In this screen, the user is asked to confirm they are not a robot
+ */
+class FtueAuthLegacyStyleCaptchaFragment @Inject constructor(
```

Comment:
a copy of https://github.com/vector-im/element-android/pull/5737/files#diff-b79246f1e3b9be4ece74ea4b504d1316f14e39c18733b5256ce41d9c6d379948R38 with a different UI style